data/Nuellity.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "Emmanuel Abhulimhen",
+  "bio": "Software Engineer targeting Javascript | React | Node.js | MongoDB | Frontend | Web3 | Python opportunities.",
+  "avatar": "https://imgur.com/a/FLuvF3r.png",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "NextJS",
+    "NodeJS",
+    "Python",
+  ],
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/emmabhu",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Nuellity",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Check out my portfolio",
+      "url": "https://portfolio-pixk.onrender.com/",
+      "icon": "FaGlobe"
+    },
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/emmanuelabhulimhen/",
+      "icon": "FaLinkedin"
+    }
```

```suggestion
    }
    ]
```
there is a closing bracket missing